modules/scan-inactive-images.adoc

```diff
@@ -15,6 +15,13 @@ You can also configure {product-title-short} to scan inactive (not deployed) ima
 
 . In the {product-title-short} portal, click *Vulnerability Management* -> *Results*.
 . Click *More Views* -> *Inactive images*.
+. Optional: To view the component and advisory data associated with a CVE, complete the following steps:
```

Comment:
Advisory can be viewed by going to cve, image or deployment details page. it is shown in three places. make this more clear